docs/mastering-plone-5/instructions_plone5/index.md

```diff
@@ -0,0 +1,382 @@
+2019-03-07 10:38:17,670 WARNI [Init:88][MainThread] Class Products.CMFFormController.ControllerValidator.ControllerValidator has a security declaration for nonexistent method 'ZPythonScriptHTML_changePrefs'
+2019-03-07 10:38:21,160 WARNI [plone.behavior:172][MainThread] Specifying 'for' in behavior 'Tiles' if no 'factory' is given has no effect and is superfluous.
+2019-03-07 10:38:22,473 WARNI [PrintingMailHost:30][MainThread] Hold on to your hats folks, I'm a-patchin'
+2019-03-07 10:38:22,474 WARNI [PrintingMailHost:124][MainThread]
+
+******************************************************************************
+
+Monkey patching MailHosts to print e-mails to the terminal.
+
+This is instead of sending them.
+
+NO MAIL WILL BE SENT FROM ZOPE AT ALL!
+
+Turn off debug mode or remove Products.PrintingMailHost from the eggs
+or remove ENABLE_PRINTING_MAILHOST from the environment variables to
+return to normal e-mail sending.
+
+See https://pypi.python.org/pypi/Products.PrintingMailHost
+
+******************************************************************************
+
+2019-03-07 10:38:22,510 INFO  [Zope:44][MainThread] Ready to handle requests
+Starting server in PID 25230.
+Serving on http://0.0.0.0:8080
+```
+
+````{note}
+In rare cases when you are using macOS with an UTF-8 character set starting Plone might fail with the following error:
+
+```text
+ValueError: unknown locale: UTF-8
+```
+````
+
+In that case you have to put the localized keyboard and language settings in the .bash_profile
+of the vagrant user to your locale (like `en_US.UTF-8` or `de_DE.UTF-8`)
+
+```shell
+export LC_ALL=en_US.UTF-8
+export LANG=en_US.UTF-8
+```
+
+Now the Zope instance we're using is running.
+You can stop the running instance anytime using {kbd}`ctrl + c`.
+
+If it doesn't, don't worry, your shell isn't blocked.
+
+Type {kbd}`reset` (even if you can't see the prompt) and press RETURN, and it should become visible again.
+
+If you point your local browser at <http://localhost:8080> you see that Plone is running in Vagrant.
+
+This works because VirtualBox forwards the port 8080 from the guest system (the vagrant Ubuntu) to the host system (your normal operating system).
+
+There is no Plone site yet - we will create one in chapter 6.
+
+The Buildout for this Plone is in a shared folder.
+This means we run it in the vagrant box from {file}`/vagrant/buildout` but we can also access it in our own operating system and use our favorite editor.
+
+You will find the directory {file}`buildout` in the directory {file}`training` that you created in the beginning
+next to {file}`Vagrantfile` and {file}`manifests`.
+
+```{note}
+The database and the python packages are not accessible in your own system since large files cannot make use of symlinks in shared folders.
```

Comment:
```suggestion
The database and the Python packages are not accessible in your own system since large files cannot make use of symlinks in shared folders.
```